### PR TITLE
SAR-10864 Update TestREADME

### PR DESCRIPTION
**Pull request by EviNikolaidou**

### CODE REVIEW

File: TestREADME.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+---
+
+1. [Setting up the Feature Switches](TestREADME.md#get-test-onlyfeature-switches)
+2. [Allocated Enrolment Feature Switch](TestREADME.md#get-enrolmentshmrc-mtd-vatvrnvatnumberusers)
+2. [Query User Call Feature Switch](TestREADME.md#post-groupsgroupidenrolmentsenrolmentkey)
+3. [Using the Allocated Enrolment Call stub](TestREADME.md#Using-the-Allocated-Enrolment-Call-stub)
+4. [Using the query user Call stub](TestREADME.md#Using-the-query-user-Call-stub)
+
+
+#### GET test-only/feature-switches
```
Comment (slavvco) [inline TestREADME.md:14]: maybe ### instead ####?

File: TestREADME.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+---
```
Comment (slavvco) [inline TestREADME.md:16]: --- should be above GET test-only/feature-switches ?

**Review by slavvco — CHANGES_REQUESTED** [review]

File: TestREADME.md (hunk 2 above)
Comment (EviNikolaidou) [reply TestREADME.md:16]: No it should be below, to underline the title. I need to add it to the other titles too

**Review by slavvco — APPROVED** [review]
Looks good

**Pull request merged**